test/e2e/kubectl/kubectl.go

```diff
@@ -1095,7 +1095,7 @@ metadata:
 		})
 	})
 
-	ginkgo.Describe("Kubectl cluster-info dump", func() {
+	ginkgo.Describe("Kubectl cluster-info dump [DisabledForLargeClusters]", func() {
```

Comment:
I would like to get rid of all DisabledForLargeClusters tests.

So if you're adding a new one, then please open an issue describing what is the problem (large memory usage) and potential ways to fix it eventually (e.g. would changing kubectl to use protobufs help?)

Reply:
Added: https://github.com/kubernetes/kubernetes/issues/94649